package.json

```diff
@@ -54,6 +54,9 @@
     "test": "NODE_ENV=test jest"
   },
   "jest": {
+    "globals": {
+      "__DEV__": true
```

Comment:
@cpojer Because I added it here.

It was unclear to me if using `var` in environment would be in the global scope but there was no question about using this config.